vitrina/projects/views.py

```diff
@@ -111,4 +117,51 @@ class ProjectHistoryView(HistoryView):
     model = Project
     detail_url_name = 'project-detail'
     history_url_name = 'project-history'
-    tabs_template_name = 'component/tabs.html'
+    tabs_template_name = 'vitrina/projects/tabs.html'
+
+
+class ProjectDatasetsView(HistoryMixin, ListView):
+    model = Dataset
+    template_name = 'vitrina/projects/datasets.html'
+    paginate_by = 20
+
+    object: Dataset
```

Comment:
```suggestion
    object: Project
```

But I would probably not use `object` name here, because this is a `MultipleObject` mixin, and `object` is from `SingleObject` mixin, so using both `model` and `object` here is very confusing.

Maybe rename `object` → `project`? To avoid confusion?